.travis.yml

```diff
@@ -11,27 +11,33 @@ os:
   - linux
   - osx
 
+sudo: required # needed for trusty
+dist: trusty # needed for Raqm
+
 before_install:
   - if [ "$TRAVIS_OS_NAME" = "linux" ];
     then
+      sudo apt-add-repository -y ppa:as-bahanta/raqm;
```

Comment:
your concern is certainly valid.  but i think that's the route we should take as long as the package is unavailable in the distro.

looking at the changes, i don't think it's that big of a deal as code is not that intrusive.